src/test/java/org/fundacionjala/sfdc/stepdefinition/LogInSteps.java

```diff
@@ -0,0 +1,43 @@
+package org.fundacionjala.sfdc.stepdefinition;
+
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import org.fundacionjala.sfdc.commons.DriverManager;
+import org.fundacionjala.sfdc.commons.PropertiesManager;
+import org.fundacionjala.sfdc.pageobjects.SalesForceLogIn;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import java.util.Properties;
+
+/**
+ * LogInSteps.java.
+ * Class for steps logIn.feature.
+ */
+public class LogInSteps {
+    private WebDriver web;
+    private Properties prop;
+    private SalesForceLogIn login;
+
+    /**
+     * Given step.
+     */
+    @Given("^Open the Firefox and launch the application enter the Username and Password and press login$")
+    public void openTheFirefoxAndLaunchTheApplicationEnterTheUsernameAndPasswordAndPressLogin() {
+        System.out.println("Open Browser and go to login page of the SalesForce and fill the username and password.");
+        web = DriverManager.getInstance().getNavigator();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2209)